cmd/fyne/internal/commands/encode.go

```diff
@@ -0,0 +1,8 @@
+package commands
+
+import "strings"
+
+func encodeXMLString(in string) string {
+	amped := strings.ReplaceAll(in, "&", "&amp;")
+	return strings.ReplaceAll(strings.ReplaceAll(amped, "<", "&lt;"), ">", "&gt;")
```

Comment:
I wonder if there perhaps isn't a better alternative already in the standard library?
[This link](https://stackoverflow.com/questions/22906722/how-to-encode-special-characters-in-xml) seems to indicate that there are quite a few special characters.

Some googling on "html escape string in Go" seems to bring this up: https://stackoverflow.com/questions/38065765/how-to-escape-the-html-using-golang.

Reply:
Using [html.EscapeString](https://pkg.go.dev/html#EscapeString) seems to be the right way to go.